charts/aws-cloud-controller-manager/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+name: aws-cloud-controller-manager
+description: This will install Cloud Controller Manager for AWS Cloud Provider
+version: 0.0.1
```

Comment:
I'd say we should at least match the major and minor app version here. Thoughts?

Reply:
I think the chart version is usually independent, isn't it?

Reply:
It is, but I personally find it less confusing if it matches the major and minor version. No strong preference here :)

Reply:
I think that would encourage it to be conflated with the app version, but also not a strong preference.

Reply:
@nckturner @ayberk Usually it is followed not with respect to the version which we are releasing. Please let me know your thoughts on this :)

Reply:
That's fine w/ me

Reply:
Resolve the conversation for now :), Have added appVersion in the chart as suggested by @ayberk. This will help the end-users to verify the version of the App I guess. Thanks